allure-results/data/behaviors.csv

```diff
@@ -0,0 +1,17 @@
+"Epic","Feature","Story","FAILED","BROKEN","PASSED","SKIPPED","UNKNOWN"
```

Comment:
do we need this file on git?

Reply:
Actually no. It won't be in the next PR.